PathPlanning/LPAStar/lpa_star.py

```diff
@@ -0,0 +1,284 @@
+"""
+Lifelong Planning A*
+author: flyingmars (github.com/2892510130)
```

Comment:
Please make a PR focusing on one issue. Do not include multiple contributions in one PR.